src/Data/Product/Properties.agda

```diff
@@ -53,6 +55,9 @@ module _ {B : A → Set b} where
 ,-injective : ∀ {a c : A} {b d : B} → (a , b) ≡ (c , d) → a ≡ c × b ≡ d
 ,-injective refl = refl , refl
 
+map-cong₂ : ∀ {f g : A → C} {h i : B → D} → f ≗ g → h ≗ i → map f h ≗ map g i
+map-cong₂ p q (x , y) = cong₂ _,_ (p x) (q y)
```

Comment:
Very minor bikeshedding but could we rename `p` and `q` to `f≗g` and `h≗i` instead?

Reply:
Sure